OpenRA.Mods.Common/Traits/Buildings/Building.cs

```diff
@@ -159,7 +159,7 @@ public Building(ActorInitializer init, BuildingInfo info)
 
 		public void Created(Actor self)
 		{
-			if (SkipMakeAnimation || !self.HasTrait<WithMakeAnimation>())
+			if (SkipMakeAnimation || (!self.HasTrait<WithMakeAnimation>() && !self.HasTrait<WithMakeAnimation>()))
```

Comment:
This is duplicating the check, probably an auto-rename left over. You should think about squashing things and getting rid of the workaround commits.